packages/test_packages_common.py

```diff
@@ -41,9 +41,9 @@ def test_parse_package(name):
 
 
 @pytest.mark.skip_refcount_check
-@pytest.mark.driver_timeout(40)
+@pytest.mark.driver_timeout(60)
 @pytest.mark.parametrize("name", registered_packages())
-def test_import(name, selenium_standalone):
+def test_import(name, selenium):
```

Comment:
Well, this in particular aims to check that run dependency are correct. if we reuse the selenium instance, we could have a case, where dependencies are incorrect but the package was loaded in a previous test and so the current test passes even when it shouldn't. 

Though yes, it's slow and a bit unfortunate.

Reply:
Maybe try reloading Pyodide rather than refreshing the page? My guess is this'll lead to trouble but it's worth trying just to see.

Reply:
Yes, I think we would eventually run into memory issues if we do that. Particularly with little RAM available in CI.

An ideal solution would to re-use the same browser profile and browser cache (including WASM compilation cache for Chrome), but I have never managed to make it work reliably when I tried in the past . By default, selenium creates a new browser profile for each new test run, with is really not great for our use case.

Reply:
> Well, this in particular aims to check that run dependency are correct. if we reuse the selenium instance, we could have a case, where dependencies are incorrect but the package was loaded in a previous test and so the current test passes even when it shouldn't.

Yeah, that could be problematic. I'll seek for a better solution.